Modules/ThirdParty/PNG/UpdateFromUpstream.sh

```diff
@@ -26,5 +26,5 @@ extract_source () {
     popd
 }
 
-source "${BASH_SOURCE%/*}/../../../Utilities/Maintenance/UpdateThirdPartyFromUpstream.sh"
+source "${BASH_SOURCE%/*}/../../../Utilities/Maintenance/update-third-party.bash"
```

Comment:
Does this work?

Reply:
After a proper update it still doesn't work. There is the follow errors during the script:
```
+ a1d11567ee...47ed77d879 upstream-itkpng -> upstream-itkpng  (forced update)
++ git merge --log -s ours --no-commit --allow-unrelated-histories upstream-itkpng
Automatic merge went well; stopped before committing as requested
++ git read-tree -u --prefix=Modules/ThirdParty/PNG/src/itkpng/ upstream-itkpng
error: Entry 'Modules/ThirdParty/PNG/src/itkpng/LICENSE' overlaps with 'Modules/ThirdParty/PNG/src/itkpng/LICENSE'.  Cannot bind.
+ rm -rf /Users/blowekamp/src/ITK/work
```

Reply:
We may need a one-time manual re-import of the existing PNG version with the new script.  I'll look at this and push an update.

Reply:
Maybe cherry-pick 7abd4e0db581e37d22fca052323d45088144078e and make another PR.

Reply:
#5054 re-imports PNG 1.6.44 with the new import script.  Once that is merged, this PR can be rebased.

Reply:
@hjmjohnson Thank your work on ITK, but please be more careful on the merge button. see the above message.

Reply:
We should have revoked our approvals during the review process.